### PR TITLE
Newsletter 13: Piston

### PR DESCRIPTION
**Pull request by bvssvni**

### CODE REVIEW

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -66,6 +66,90 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+One research project with Nano-ECS is to prototype a UI framework
+for Rust with a UI editor (not open sourced yet).
+This project uses Piston-Graphics by default,
+but can generate draw commands for processing by other 2D APIs.
+It is also possible to override rendering of widgets for
+custom looks with Piston-Graphics, which is often useful in gamedev.
+Recently, this project has gotten to a place where
+[tree-view interaction](https://tinyurl.com/y63f5xkv) is working.
+
+[Piston Discord Channel]: https://t.co/8YOj3auDr9?amp=1
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:144]: These [reference-style links](https://spec.commonmark.org/0.29/#reference-link) don't seem to be used anywhere - did you mean to use [inline links](https://spec.commonmark.org/0.29/#inline-link) instead?

Comment (bvssvni) [reply content/posts/newsletter-013/index.md:143]: @17cupsofcoffee Thank you! I hope I got this right this time.

Comment (17cupsofcoffee) [reply content/posts/newsletter-013/index.md:143]: This looks better :+1:

These links still aren't used anywhere in the post, though: 
* `[@PistonDeveloper at Twitter]`
* `[Piston]`
* `[Dyon]`
* `[Piston-Graphics]`

If you want those URLs to be accessible, you'll need to either reference those links somewhere in the text or replace them with inline links. Markdown's link syntax is a bit inscrutable, if you get stuck let me know what it is you're trying to achieve and I can help work out the markup for it :+1:

FYI: You can click this icon on the Files Changed page of the PR to preview your changes - that's how I've been checking what the rendered post looks like: 
image

Comment (bvssvni) [reply content/posts/newsletter-013/index.md:213]: @17cupsofcoffee Thank you! I've fixed the link and resolved the merge conflict.

**Review by 17cupsofcoffee — APPROVED** [review]
Looks good, thank you for making the changes!

**Comment by ozkriff** [comment]
The Discord link doesn't seem to work for me (both https://discord.gg/XNpF2j and twitter-shortened versions):

image

**Review by ozkriff — CHANGES_REQUESTED** [review]
lgtm (added a logo and made a few small fmt tweaks)

The Discord link needs to be fixed though.

**Comment by bvssvni** [comment]
@ozkriff Thanks! Discord link is fixed.

**Review by ozkriff — APPROVED** [review]
:+1:

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**